vlog4j-core/src/main/java/org/semanticweb/vlog4j/core/reasoner/implementation/VLogReasoner.java

```diff
@@ -594,6 +594,32 @@ public QueryResultIterator answerQuery(PositiveLiteral query, boolean includeBla
 		return new VLogQueryResultIterator(stringQueryResultIterator, this.correctness);
 	}
 
+	@Override
+	public long queryAnswerSize(PositiveLiteral query) {
+		return queryAnswerSize(query, true);
+	}
+
+	@Override
+	public long queryAnswerSize(PositiveLiteral query, boolean includeNulls) {
+		validateNotClosed();
+		Validate.notNull(query, "Query atom must not be null!");

```

Comment:
the method should behave like `answerQuery`, i.e. throw an exception if the KB has not been loaded.